src/components/tooltip/tooltip.ts

```diff
@@ -43,6 +43,7 @@ export class MdTooltip {
   }
 
   private _overlayRef: OverlayRef;
+  private _tooltipRef: TooltipComponent;
```

Comment:
I didn't think of this in the first PR, but you should rename `TooltipComponent` to `TooltipContainer`

(and then just call this member `_tooltipContainer`)